dadguide/models/base_model.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+
+class BaseModel(object):
+
+    def generate_log(self):
+        raise NotImplementedError
+
+    def to_json(self):
```

Comment:
i still think you should make `generate_log` -> `to_json`, since `json.dumps` is really "to_json_string"

Reply:
`to_dict` is also acceptable

Reply:
ok changed to `to_dict`